language-plutus-core/generators/Language/PlutusCore/Generators/Internal/TypeEvalCheck.hs

```diff
@@ -103,7 +101,7 @@ typeEvalCheckBy eval (TermOf term x) = TermOf term <$> do
 unsafeTypeEvalCheck
     :: forall a. KnownType a => TermOf a -> TermOf EvaluationResultDef
 unsafeTypeEvalCheck termOfTbv = do
-    let errOrRes = typeEvalCheckBy (pureTry @CkMachineException . evaluateCk) termOfTbv
+    let errOrRes = typeEvalCheckBy (evaluateCek mempty) termOfTbv
```

Comment:
Now that we have the CEK machine in `language-plutus-core`, we can use it for tests instead of the CK machine.